demo-react/src/components/Avatar.tsx

```diff
@@ -0,0 +1,89 @@
+import cn from 'classnames';
+import { AvatarInfo } from './AvatarInfo';
+
+export interface AvatarProps {
+  isCurrent?: boolean;
+  name: string;
+  isActive?: boolean;
+  color?: string;
+}
+
+export const Avatar = ({
+  isCurrent = false,
```

Comment:
I think it would be nice to use `isSelf` here as that aligns with the Spaces terminology

Reply:
done ✅